simulator/util/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

```diff
@@ -40,20 +40,20 @@
   </group>
 
   <!-- preception module -->
-  <group>
+  <group ns="perception">
```

Comment:
We cannot use `ns` with `group`. Instead, we are using `push-ros-namespace` which is there already.

Reply:
@mitsudome-r  Thank you for the comment! I commit to removing unnecessary 'ns'